lib/membrane_raw_audio/sample_format.ex

```diff
@@ -134,8 +134,11 @@ defmodule Membrane.RawAudio.SampleFormat do
 
     endianness =
       case size do
-        8 -> :any
-        _ -> @sample_endiannesses |> BiMap.get_key(serialized_format &&& @sample_endianness)
+        8 ->
+          :any
+
+        _otherwise ->
+          @sample_endiannesses |> BiMap.get_key(serialized_format &&& @sample_endianness)
```

Comment:
'Brain.exe has stopped working'

These constants give me a headache. Maybe we could change them to `endianness_mapping` and `endianness_bitmask`